modules/luci-mod-dsl/root/usr/share/luci/menu.d/luci-mod-status-dsl.json

```
@@ -0,0 +1,14 @@
+{
+         "admin/status/dsl-spectrum": {
+                 "title": "DSL line spectrum",
+                 "order": 7,
+                 "action": {
+                         "type": "view",
+                         "path": "status/dsl/dsl-spectrum"
+                 },
+                 "depends": {
+                         "acl": [ "luci-mod-status-dsl" ],
+                         "uci": { "network": { "@dsl": true } }
```

You probably meant:
```suggestion
                         "uci": { "network": { "dsl": true } }
```
At least this fixes it for me on the unmerged VR11 support